pkg/reconciler/openshift/openshift.go

```diff
@@ -83,6 +87,24 @@ func getPolicyRuleForRedisHa() rbacv1.PolicyRule {
 	}
 }
 
+func getPolicyRuleForApplicationController() []rbacv1.PolicyRule {
+	return []rbacv1.PolicyRule{
```

Comment:
I don't see why this is openshift specific and can't just go in policyRuleForApplicationController().

Also, I tested the upgrade and rbac is not fixed and existing instances still have permission problems after grading.

Reply:
By the default the rule provided in `policyRuleForApplicationController()` is for the community operator which is to provide the `"*"` admin access to all api groups and resources. 
But we don't want that for our Openshift Deployments. So we aggregate read permissions for all resources with Cluster Role Admin. This rule provides read access to all api groups and resources.

Reply:
Why community operator grant omni permission?